README.md

```diff
@@ -176,6 +176,15 @@ mappings:
     provider: "$2"
     outcome: "$3"
     job: "${1}_server"
+  quantiles:
+    - quantile: 0.99
+      error: 0.001
+    - quantile: 0.95
+      error: 0.01
+    - quantile: 0.9
+      error: 0.05
+    - quantile: 0.5
+      error: 0.0.05
```

Comment:
is this an extra dot here? Could you also add some explanation of what this means? You can link to the [documentation](https://prometheus.io/docs/practices/histograms/#quantiles) for details on quantiles but please do mention that this is what is being configured here.

Reply:
Done.